### PR TITLE
chore(update documentation): Added a hosted PDF and updated docs link

### PR DESCRIPTION
**Pull request by nr-jbiggley**
# Summary

Updated the documentation link to refer to a PDF doc that is hosted in the Quickstart folder.

### CODE REVIEW

**Comment by nr-jbiggley** [comment]
Also added an update to the dashboard.

File: quickstarts/gigamon/config.yml [hunk 1]
```diff
@@ -38,8 +38,11 @@ installPlans:
 
 documentation:
   - name: Gigamon Cloud Suite Installation
-    url: https://www.gigamon.com/lp/free-trial/request-live-cloud-demo.html
-    description: The link refers to setting up demo for Cloud Suite Solution required to deploy the NewRelic Integration.
+    url: https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/gigamon/Gigamon New Relic Integration.pdf
+    description: An overview of the architecture and deployment methodology for Gigamon Hawk integrated with New Relic
```
Comment (jpvajda) [inline quickstarts/gigamon/config.yml:42]:
```suggestion
    description: | 
        An overview of the architecture and deployment methodology for Gigamon Hawk integrated with New Relic
```

File: quickstarts/gigamon/config.yml [hunk 2]
```diff
@@ -38,8 +38,11 @@ installPlans:
 
 documentation:
   - name: Gigamon Cloud Suite Installation
-    url: https://www.gigamon.com/lp/free-trial/request-live-cloud-demo.html
-    description: The link refers to setting up demo for Cloud Suite Solution required to deploy the NewRelic Integration.
+    url: https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/gigamon/Gigamon New Relic Integration.pdf
```
Comment (jpvajda) [inline quickstarts/gigamon/config.yml:41]:
```suggestion
    url: >- 
       https://github.com/newrelic/newrelic-quickstarts/tree/main/quickstarts/gigamon/Gigamon New Relic Integration.pdf
```

**Review by jpvajda — CHANGES_REQUESTED** [review]
I made some suggestions on the documentation URL / description.

**Comment by nr-jbiggley** [comment]
> I made some suggestions on the documentation URL / description.

Thanks for the feedback, @jpvajda

**Review by jpvajda — APPROVED** [review]

**Pull request merged**